### DISCUSSION TITLE
Block JUST creeper explosions in claims

### DISCUSSION THREAD

**Discussion by JHarris12345** [GriefPrevention/GriefPrevention#727] [Ideas]
As a server owner I would like to see a config option that blocks just creeper explosions in claims rather than all explosions. It would help my server to disable creepers being led into claims to ruin them whilst still allowing users of the claims to use tnt etc

Optional questions:

Is your feature request related to a problem? Please describe.
No

Describe alternatives you've considered
There arent many for this

**Comment by RoboMWM** [comment]
Any concern for TNT cannons or adjacent TNT explosions?

Claim owners can use `/claimexplosions` to use TNT.
